src/test/java/lotto/domain/BonusNumberTest.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import lotto.domain.lotto.LottoNumber;
+import lotto.exception.InValidBonusNumberException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class BonusNumberTest {
+    @Test
+    @DisplayName("BonusNumber 인스턴스 생성시 bonusNumber가 winningNumbers와 겹치는 경우 ")
+    void testInstanceCreationFailCase() {
+        WinningNumbers winningNumbers = WinningNumbers.valueOf(new HashSet<>(Arrays.asList(
+                LottoNumber.valueOf(1), LottoNumber.valueOf(2), LottoNumber.valueOf(3),
+                LottoNumber.valueOf(4), LottoNumber.valueOf(5), LottoNumber.valueOf(6))));
```

Comment:
Nit: 혹시 테스트를 작성하시면서 `WinningNumbers` 인스턴스를 만들기 쉽지 않다는 느낌이 들진 않으셨나요?
좀 더 쉽게 인스턴스를 생성할 수 있도록 해보면 어떨까요?
```suggestion
        WinningNumbers winningNumbers = WinningNumbers.valueOf(Set.of(1, 2, 3, 4, 5, 6));
```